tests/steps/other_v2_steps.py

```diff
@@ -1469,11 +1473,42 @@ def simulate_atc_failure(context, group, path, message):
             ]
         ]
     )
-    assert resp.would_succeed is False
     assert fail_path == path
     assert message in resp.failure_message
 
 
+@when("I make a new simulate request.")
+def make_simulate_request(context):
+    context.simulate_request = SimulateRequest(txn_groups=[])
+
+
+@then("I lift log limits on that simulate request.")
+def lift_log_limits_in_request(context):
+    context.simulate_request.lift_log_limits = True
+
+
+@then("I attach the simulate request to simulate the transaction group.")
+def attach_sim_request_to_txn_group_simulation(context):
+    context.simulate_response = (
+        context.atomic_transaction_composer.simulate_with_request(
+            context.app_acl, context.simulate_request
+        )
+    )
+
+
+@then('the simulation with "{power}" should not have failure message.')
+def power_pack_simulation_should_pass(context, power: str):
```

Comment:
This step is checking two different things (power pack in response and no failure message), perhaps break this up into 2 different ones? I'd rather have a new step for each power pack than trying to fit all the future ones into here.